requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 pylint
 radon
 # for build_htmlhelp command
-sphinx>=1.5
\ No newline at end of file
+sphinx>=1.5
+recommonmark
```

Comment:
`requirements-dev.txt` requires a newline at the end (for the benefit of our travis ci scripts).